python/tank/bootstrap/configuration_writer.py

```diff
@@ -351,6 +350,10 @@ def write_config_info_file(self, config_descriptor):
             # write yaml
             yaml.safe_dump(metadata, fh)
 
+        core_location_ini_file = os.path.join(self._path.current_os, "cache", "core_location.ini")
```

Comment:
This hasn't been implemented, but this file can be reused from the tank command to find the core.